### PR TITLE
VAR-94 | Add specific terms in new reservation form

### PR DESCRIPTION
**Pull request by Rasbin**
`Specific terms` should be added under new reservation form as a plain text below terms checkbox and above save button.

### CODE REVIEW

File: app/pages/reservation/reservation-information/ReservationInformationForm.js [hunk 1]
```diff
@@ -285,7 +285,7 @@ class UnconnectedReservationInformationForm extends Component {
               t('common.billingEmailAddressLabel'),
             )
           }
-          <h2 className="app-ReservationPage__title">{t('ReservationInformationForm.eventInformationTitle')}</h2>
+          <h2 className="app-renderTermsField">{t('ReservationInformationForm.eventInformationTitle')}</h2>
```
Comment (jimppa) [inline app/pages/reservation/reservation-information/ReservationInformationForm.js]: Why did you change the className here?

Comment (Rasbin) [reply app/pages/reservation/reservation-information/ReservationInformationForm.js]: Mistakenly got changed. Reverted back in latest commit.
